src/views/BottomTabBar.js

```diff
@@ -220,9 +226,14 @@ class TabBarBottom extends React.Component<Props> {
         {routes.map((route, index) => {
           const focused = index === navigation.state.index;
           const scene = { route, focused };
-          const accessibilityLabel = this.props.getAccessibilityLabel({
-            route,
-          });
+          const accessibilityLabel = this.props.getAccessibilityLabel({ route });
+          const accessibilityRole = this.props.getAccessibilityRole({ route }) || 'button';
+
+          let accessibilityStates = this.props.getAccessibilityStates({ route });
+          if (!accessibilityStates) {
+            accessibilityStates = focused ? ['selected'] : [];
```

Comment:
What other accessibility states instead of selected/not selected do you have in mind?

Reply:
`selected` and `disabled` are the only options allowed through the API, but this library would not really know if a tab is disabled (perhaps for an app's custom permissions?). If someone had a disabled tab, they could override `getAccessibilityStates` and return `['disabled']` or even `['selected', 'disabled']`.